packages/design-system/src/components/Dropdown/Dropdown.tsx

```diff
@@ -15,12 +15,12 @@ type DropdownButtonType = Omit<ClickableProps, 'children' | 'as'> & {
 	onClick: () => void;
 	icon?: DeprecatedIconNames;
 	type: 'button';
-};
+} & Partial<DataAttributes>;
```

Comment:
Just a tough, i see the Partial<DataAttributes> everywhere but nothing it required in this Type, i don't know, should it work the same without the partial ?

```suggestion
} & DataAttributes;
```